woocommerce/blocks-template
# Add new file to house the integration class and prevent a fatal error

This is required to stop a fatal error happening when `Automattic\WooCommerce\Blocks\Integrations\IntegrationInterface` is used before it's ready.

### Testing
To test this:
1. run `npx @wordpress/create-block blocks-template my-extension` (replace `my-extension` with any other slug)
2.  Move the resulting folder to `wp-content/plugins`
3. Load the Cart or Checkout block and you should see  `Data passed to this component: This is some example data from the server` under the cart total
